### PR TITLE
docs(router): fix broken link

### LINKED ISSUE #14430: docs(router): router overview points to broken site
**I'm submitting a ...**  (check one with "x")
```
[ x] bug report => search github for a similar issue or PR before submitting
```

**Current behavior**
https://github.com/angular/angular/blob/master/modules/%40angular/router/README.md points to "Read the overview of the Router [here](https://victorsavkin.com/post/145672529346/angular-router)". But visiting the link gives `Urgent DNS Migration Required` because the Firebase Hosting site hasn't been migrated to support HTTP/2.

Even trying https://victorsavkin.com fails due to this issue with Firebase Hosting. More details on the fix here: https://firebase.googleblog.com/2016/09/http2-comes-to-firebase-hosting.html

**Expected behavior**
Visiting the link to the router overview would load the overview guide.

**Minimal reproduction of the problem with instructions**
Click the link in the README above.

**What is the motivation / use case for changing the behavior?**
To better support people who are looking for router docs beyond what is on angular.io.

**Please tell us about your environment:**
OX S, IntelliJ IDEA

* **Angular version:** 2.4.7
* **Angular router version:** 3.4.7

* **Browser:** all
 
* **Language:** all

* **Node (for AoT issues):** `node --version` = 6.9.2

### PR DESCRIPTION
**Pull request by DzmitryShylovich**
Closes #14430

### CODE REVIEW

File: modules/@angular/router/README.md [hunk 1]
```diff
@@ -6,11 +6,14 @@ Managing state transitions is one of the hardest parts of building applications.
...
 
 ## Overview
-Read the overview of the Router [here](http://victorsavkin.com/post/145672529346/angular-router).
+Read the overview of the Router [here](https://vsavkin.com/angular-2-router-d9e30599f9ea#.lcmgpt8p7).
 
 ## Guide
 Read the dev guide [here](https://angular.io/docs/ts/latest/guide/router.html).
 
+## Book
+Read the book [here](https://leanpub.com/router).
```
Comment (vicb) [inline modules/@angular/router/README.md]: IMO you should mention:
- this is a paid book,
- written by victor savkin the architect of the router.

File: modules/@angular/router/README.md [hunk 2]
```diff
@@ -6,11 +6,14 @@ Managing state transitions is one of the hardest parts of building applications.
 The Angular router is designed to solve these problems. Using the router, you can declaratively specify application state, manage state transitions while taking care of the URL, and load components on demand.
 
 ## Overview
-Read the overview of the Router [here](http://victorsavkin.com/post/145672529346/angular-router).
+Read the overview of the Router [here](https://vsavkin.com/angular-2-router-d9e30599f9ea#.lcmgpt8p7).
```
Comment (vicb) [inline modules/@angular/router/README.md]: remove the # part

**Comment by DzmitryShylovich** [comment]
@vicb updated

**Review by vicb — APPROVED** [review]

**Comment by IgorMinar** [comment]
hi @DzmitryShylovich 

README files are not an appropriate place for advertising paid services, and we try to avoid favoritism. We will not add a paid book link to the readme.

Can you please change the PR to update the blog post link but not add the other one? thanks!

**Review by IgorMinar — CHANGES_REQUESTED** [review]
hi @DzmitryShylovich

README files are not an appropriate place for advertising paid services, and we try to avoid favoritism. We will not add a paid book link to the readme.

Can you please change the PR to update the blog post link but not add the other one? thanks!

**Comment by DzmitryShylovich** [comment]
@IgorMinar done

**Review by IgorMinar — APPROVED** [review]

**Comment by IgorMinar** [comment]
thanks @DzmitryShylovich

**Pull request merged**